common/oatbox/filesystem/utils/FileSystemWrapperTrait.php

```diff
@@ -33,7 +33,7 @@ trait FileSystemWrapperTrait
      * @see \League\Flysystem\FilesystemInterface::has()
      */
     public function has($path) {
-        return $this->getFileSystem()->has($path);
+        return $this->getFileSystem()->has($this->getFullPath($path));
```

Comment:
This trait is a full homogeneous decorator. Can't we use the original class with just a modifed parameter (getFullPath($path))?